scss/_common.scss

```diff
@@ -6,6 +6,7 @@ body {
 }
 
 html {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```

Comment:
`system-ui` is the standerdized (working Draft) keyword:
```suggestion
  font-family: system-ui, -apple-system, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif;
```
see https://caniuse.com/#feat=font-family-system-ui

Reply:
I just used what we use in Bootstrap, minus the Emoji fonts.

Reply:
nvm, not sure if https://infinnie.github.io/blog/2017/systemui.html still applies